testing.sagebrush.master/Stepwat_Inputs/Input/sxw/Input/siteparam.in

```diff
@@ -86,19 +110,19 @@ RCP85
 
 Campbell1974        # Specify soil water retention curve
 Cosby1984AndOthers  # Specify pedotransfer function
-                    # (if not implemented, then provide SWRC parameters via "swrc_params.in")
+                    #   * if not implemented, then provide SWRC parameters
+                    #     as inputs via "swrc_params.in" and/or "swrcp.nc")
 
-0     # Has SWRC parameters (see `has_swrcp`)?
-      #    0: Estimate with specified pedotransfer function
-      #    1: Use values from "swrc_params.in"
+0                   # Are SWRC parameters for the mineral soil component
+                    # provided as inputs (see `has_swrcp`)?
+                    #   0: Estimate with specified pedotransfer function
+                    #   1: Use inputs from "swrc_params.in" and/or "swrcp.nc"
 
 
-# ---- Transpiration regions ----
+#---- Transpiration regions
 # ndx  : 1=shallow, 2=medium, 3=deep, 4=very deep
-# layer: deepest layer number of the region.
-#        Layers are defined in soils.in.
-# ndx    layer
-	1		2
-	2		4
-	3		6
-#	4		8
+# region depth (cm): depth from soil surface to bottom of a transpiration region
+#	ndx	depth (cm)
+	1	20
+	2	40
+	3	100
```

Comment:
transpiration region 3 was set to soil layer 6 which translates to 80 cm depth (and not 100 cm)